src/one_py_sdk/operations/spreadsheet.py

```diff
@@ -124,7 +206,7 @@ def __getRows(self, plantId, wsType, startRow=None, endRow=None, columns=None, v
                    "Accept": "application/x-protobuf"}
         response = DeserializeResponse(requests.get(url, headers=headers))
```

Comment:
I wonder if requests.get could have a default to set the header info on the request by just passing in the Auth Token.  Are all the other headers the same?